test/unit/fetchcontent_dependency/CMakeLists.txt.in

```diff
@@ -0,0 +1,38 @@
+# ~~~
+#    ┌────────────────────────┐
+#    │ FetchContentDependency │
+#    └─────┬────────────┬─────┘
+#          │1.          │3.
+#          │            │
+# ┌────────▼────┐   ┌───▼─────────┐
+# │ Dependency  ├───►  Fibonacci  │
+# └─────────────┘2. └─────────────┘
+#
+# 1. Add Project with CPMAddPackage
+# 2. Dependency will add Fibonacci with FetchContent
+# 3. Our project add Fibonacci with CPMAddPackage
+# ~~~
```

Comment:
Love the documentation of the test case!

Reply:
I used https://asciiflow.com/, very cool to embed little schema into code. And way better than lots of text.